### PR TITLE
Prepare 1.8.0 release

### LINKED ISSUE #1303: Add and communicate the CHANGELOG for 1.8.0 so others can update the Helm charts

### PR DESCRIPTION
**Pull request by sergei-maertens**
Fixes #1303

**Changes**

- [x] Update changelog
- [x] Bump version number
- [x] Check API specs for equivalency
- [x] Check translations
- [ ] Tag release commit
- [ ] Announce release
    - commonground.nl
    - Slack
    - Mailing list

### CODE REVIEW

File: src/openzaak/components/documenten/openapi.yaml [hunk 1]
```diff
@@ -426,17 +426,6 @@ paths:
...
-        in: query
-        description: Het (automatische) versienummer van het INFORMATIEOBJECT.
-        schema:
-          type: integer
-      - name: registratieOp
-        in: query
-        description: Een datumtijd in ISO8601 formaat. De versie van het INFORMATIEOBJECT
-          die qua `begin_registratie` het kortst hiervoor zit wordt opgehaald.
-        schema:
-          type: string
```
Comment (sergei-maertens) [inline src/openzaak/components/documenten/openapi.yaml:439]: this doesn't look right

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,9 +1,147 @@
...
+**New features**
+
+* [#1218] `Zaken API 1.2 <https://github.com/VNG-Realisatie/zaken-api/blob/master/CHANGELOG.rst>`_
+  features implemented
+
+    - ``ZaakObject.objectTypeOverigeDefinitie`` which can refer to object type and
+      object registrations not part of (existing) standards while ensuring strict
+      schema validation / information for clients to visualize the data.
+
+    - added PUT, PATCH and DELETE operations to ``ZaakEigenschap`` and ``ZaakObject``
```
Comment (joeribekker) [inline CHANGELOG.rst:25]:
```suggestion
    - Added PUT, PATCH and DELETE operations to ``ZaakEigenschap`` and ``ZaakObject``
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,9 +1,147 @@
...
+* [#1215] Added ``bin/dump_configuration.sh`` script to dump the runtime configuration
+  which can then be loaded into another instance.
+* [#669] Re-implemented the ``setup_configuration`` management command:
+
+    - Added extensive command line self-documentation (available via ``--help`` flag)
+    - Command actions now self-test their outcome and report problems
+    - Command can be run headless for fully automated Open Zaak installations (
+      deployment + runtime configuration)
+
+* [#1280] allow providing the ``ENVIRONMENT`` via envvar to Sentry
```
Comment (joeribekker) [inline CHANGELOG.rst:62]:
```suggestion
* [#1280] Allow providing the ``ENVIRONMENT`` via envvar to Sentry
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,9 +1,147 @@
...
+* [#1228] return null for empty verlenging information instead of object with empty fields
+* [#1247] Fixed visual regression hiding the datepicker calendar in the admin
+* [#1198] Fixed broken ordering filter in the ``zaak_zoek`` operations
+* [#1264] Fixed saving einddatum for published zaaktypen
+* [#621] Added envvar support for the ``NOTIFICATIONS_DISABLED`` configuration parameter.
+  Note that disabling notifications makes you *not compliant* with the upstream standard.
+* Fixed crash for audittrail representation generation exceeding maximum allowed length
+* The admin index fixture is now loaded after every migrate action, fixing missing menu
+  entries in upgraded installations.
+* [#1275] fixed publishing of objects with duration widgets via the admin
```
Comment (joeribekker) [inline CHANGELOG.rst:85]:
```suggestion
* [#1275] Fixed publishing of objects with duration widgets via the admin
```

**Review by joeribekker — APPROVED** [review]
Textual things only, otherwise looks good. Approving but would be nice to get the textual things fixed.

**Pull request merged**